### ISSUE TITLE
AE: Sprint 41

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#19720]
## Priorities
1.  MyVA: Usability testing prep review on Friday, begin recruiting, start research if possible
2. MyVA: Provide all copy updates to FE
3. MyVA: Continue FE build
4. DD4edu: Staging review on Tuesday, UAT research plan, convo guide, tickets from UAT, UAT go/no-go
5. DD4edu: Conduct Contact Center Review
6. Notification Preferences: Finish BE integration
7.  DD4edu: Direct deposit email change (GovDelivery -> VANotify)
8. Profile save errors

## Profile
**BE**
- [ ] Are we sending deprecated IDs to VA Profile? #17739
- [x] Log when users have a mvi_response.status of OK then return a va profile MVI201 error #18349

## My VA redesign
Hold usability review, begin recruiting for research, and start research if possible.

**PM**
- [x] MyVA 2.0 Usability Testing Prep #19389, 2:00pm ET Friday, Feb 12th
- [x] Add in research plan, convo guide, clickable prototype prior to meeting 

**Design**
- [x] MyVA 2.0 copy #18991 
- [x] [Design] MyVA: Copy updates #19165 
- [x] [Design] Create spreadsheet to track feedback from usability testing (#20104)
- [x] [Design] MyVA 2.0: Create copy updates documentation and provide to FE #19723
- [x] [Design] MyVA: Create clickable prototype for research #19164 
- [x] [Design] MyVA 2.0 Convo guide for usability testing of finalized designs #19121
- [x] [Design] MyVA 2.0: Recruit for research #19724
- [ ] [Design] MyVA 2.0: Conduct research #19725 (**Stretch goal**: We will likely only start sessions, not finish them)
- [x] Add My VA to unauth nav and add sign-in flow to prototype #19356
- [x] We need an "application has expired" state #19702

**FE** 
- [x] [FE] Export a reusable loading spinner from `RequiredLoginView` #19511
- [x] [FE] Make reusable navigation action/CTA button #18936
- [x] [FE] Build the Benefits You Might Be Interested section #18947
- [x] Improve recording of VA Profile transaction creation errors #19754
- [ ] [FE] Load Health Care data and wire it up to the Health Care widget #19743

## Direct deposit for edu
**PM**
- [x] Staging Review: DD4edu #19281 (Tuesday 2/16 at 11:00 am)
- [ ] [PM] DD4edu: Conduct Contact Center review #19734 (submitted CC review request ticket:#19899 on 2/12)

**FE**
- [x] 508-defect-1 ❗Launchblocker [AXE-CORE, COGNITION, SEMANTIC MARKUP]: Form fields and their unique IDs MUST NOT be duplicated on the page #19594
- [x] 508-defect-1 ❗ Launchblocker [AXE-CORE]: Error id attribute value must be unique #19790
- [ ] (Content blocker) 508-defect-2 [COGNITION]: H1 should match or be included in the page title #19552
- [ ] (Content blocker)  508-defect-2 [COGNITION, SEMANTIC MARKUP]: Buttons MUST have descriptive and accessible labels #19570

**Design**(Jim Adams)
- [ ] [Design] DD4edu: Create UAT Research plan #19728
- [ ] [Design] DD4edu: Create UAT Conversation Guide #19729

**BE**
- [ ] Move direct deposit emails from GovDelivery > VANotify #19233

## VA Profile Notification Preferences
Finish backend build if possible (pending Lihan getting what he needs from Terry)

**BE**
BE implementation blocked until Lihan gets what he needs from Terry

## Unplanned
- [x] [BE] Regression testing for direct deposit in staging for EVSS R17.12 release #19753
- [ ] 508-defect-1 ❗ Launchblocker [SCREENREADER]: An equivalent alternative MUST be provided for the image of the check #19611 (waiting on validation)
- [x] Content Feedback - Staging review - EDU Direct Deposit #19963 (waiting on validation)
- [x] Add real copy to "Benefits you might be interested in" #20158
- [ ] 508-defect-2 [COGNITION]: Heading levels SHOULD increase by one in a logical order #19764
